modules/warc/index.html

```diff
@@ -0,0 +1,113 @@
+---
+title: WARC-kb Module
+---
+<!DOCTYPE html>
+<html lang="en">
+{% include header.html %}
+<body role="document">
+
+{% include navbar.html nav=site.data.navbar %}
+<div class="container" role="main">
+<h1>WARC-kb Module</h1>
+<a class="name" name="introduction">
+<h2>1 Introduction</h2>
+</a>
+<p>
+  The WARC-kb module recognizes and validates the WARC (Web ARChive) format.
+  [<a href="/references#warciso">WARC</a>]. It only validates the WARC file format and WARC headers, not the actual payload of the WARC records.
```

Comment:
```suggestion
  [<a href="/references#warc">WARC</a>]. It only validates the WARC file format and WARC headers, not the actual payload of the WARC records.
```
Use here the link to the warc specification which can be directly used (as opposed to the ISO standard)

Reply:
Hi Thomas,

Thanks for the feedback! I'm was doubting about this myself. But my reasoning eventually was, that the general links (like this example) point to the ISO standard and specific links (for example the file extension information) to the publicly available specs.

I also point to the 2009 version (ISO28500:2009) while there is a newer version of 2017, but the module documentation points to the 2009 version so that's why I reference it.

So @tledoux do you want me to change this? @carlwilson do you have an opinion about this (or somebody else).